### ISSUE TITLE
Inquiry about difference in tool results

### ISSUE THREAD

**Issue by lw78943** [KosinskiLab/AlphaPulldown#203]
Dear AlphaPulldown team,

Thank you for introducing this fantastic tool to us. I have been enjoying its features, especially the four models!  I have tried to run it following the instructions of example 1, but my results differ from your screenshot. Here is my csv table:
[predictions_with_good_interpae_test.csv](https://github.com/KosinskiLab/AlphaPulldown/files/13355292/predictions_with_good_interpae_test.csv)
In my csv table, for P78344_and_Q7L2H7, the iptm is 0.1822, while your iptm value is 0.819-a notable distinction. Any insights or thoughts on why this variation might occur?

Here is my script for your reference.
version: 0.30.7
1st step:
create_individual_features.py \
  --fasta_paths=baits.fasta,example_1_sequences_shorter.fasta \
  --data_dir=/reference/data/alphafold/2.3.0/ \
  --save_msa_files=False \
  --output_dir=/project/alphaPulldown/test/output/features \
  --use_precomputed_msas=False \
  --max_template_date=2050-01-01 \
  --skip_existing=True \
  --seq_index=$SLURM_ARRAY_TASK_ID

2nd step:

module load cuda/11.2.1
module load cudnn/8.1.1

conda activate AlphaPulldown

MAXRAM=$(echo `ulimit -m` '/ 1024.0'|bc)
GPUMEM=`nvidia-smi --query-gpu=memory.total --format=csv,noheader,nounits|tail -1`
export XLA_PYTHON_CLIENT_MEM_FRACTION=`echo "scale=3;$MAXRAM / $GPUMEM"|bc`
export TF_FORCE_UNIFIED_MEMORY='1'

run_multimer_jobs.py --mode=pulldown \
    --num_cycle=3 \
    --num_predictions_per_model=1 \
    --output_path=/project/alphaPulldown/test/output/models \
    --data_dir=/reference/data/alphafold/2.3.0/ \
    --protein_lists=baits.txt,candidates_shorter.txt \
    --monomer_objects_dir=/project/alphaPulldown/test/output/features \
    --job_index=$SLURM_ARRAY_TASK_ID

3rd step:
create_notebook.py --output_dir=./ --cutoff 1000

singularity exec \
    --no-home \
    --bind /project/alphaPulldown/test/output/models:/mnt \
    /project/alphaPulldown/alpha-analysis.sif \
    run_get_good_pae.sh \
    --output_dir=/mnt \
    --cutoff=1000

**Comment by lw78943** [comment]
Hi Dingquan,

Thanks for your prompt response. I currently have access to two HPCCs, and I have executed AlphaPulldown using the same script on both. However, the results differ from my previous version, leading to confusion. 
[predictions_with_good_interpae (1).csv](https://github.com/KosinskiLab/AlphaPulldown/files/13379297/predictions_with_good_interpae.1.csv)
Could you offer insights into potential reasons for these differences? BTW, could you share your results for examples?

Additionally, I am uncertain about interpreting the results and determining which pairs of proteins are interacting. As a novice in AlphaPulldown, any guidance on result interpretation would be helpful. I apologize for the multitude of questions, and I appreciate your assistance and support.

Bests,

Li

**Comment by dingquanyu** [comment]
Hi Li,

Sorry for the confusion and my results in the screenshot was done with the first release of AlphaFold Multimer, which was quite different from the current version so I don't think the results from that will be a good reference. I've checked the results you sent and I'm sorry that they are not identical. However, AlphaFold will yield different results everytime if you provide different random seeds for it. Besides, I'm not sure if your 2 clusters have different computing resources. e.g. different GPU cards or versions of CUDA etc. as they can all contribute to the difference in numerical accuracy. 

In many other applications of AlphaFold Multimer, we saw the users sample a large number of models by giving it multiple random seed, MSA depths, etc then to get an ensemble out of this. Increasing the number of sampling might help with consistency 

With this being said, I believe the results from the 2 csv tables are not like contradicting with each other?  The scores vary but if a model is not perfect in one of the table, it wouldn't shop up as the top in another. I think the exception is P78344_and_Q9Y5M8. It was tricky as it sits close to the 0.6 cutoff and AlphaFold was already not confidence in this case so iptm scores vary between 0.4 and 0.6. If you'd like to improve the modelling of this case or anything similar, perhaps increasing the number of sampling, increasing the number of recycles, trying different MSA depths etc. would give you a more concrete prediction.

Yours
Dingquan
